go/host/host.go

```diff
@@ -716,6 +699,47 @@ func (a *Node) monitorBlocks() {
 	subs.Unsubscribe()
 }
 
+func (a *Node) catchupMissedBlocks(lastKnownBlkHash gethcommon.Hash) {
+	var lastBlkNumber *big.Int
+	var reingestBlocks []*types.Block
+
+	// get the tip block
```

Comment:
'tip block' isn't super clear, I'd say 'last processed block' or 'last ingested block' maybe?

Reply:
I was trying to avoid that, because it gets confusing. 

The last canonical block is usually called the tip of the block chain. Changing to that, wdyt ?

Reply:
```suggestion
	// get the blockchain tip block
```